lib/ansible/plugins/inventory/vultr.py

```diff
@@ -0,0 +1,99 @@
+# (c) 2018, Yanis Guenane <yanis+ansible@guenane.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: vultr
+    plugin_type: inventory
+    author:
+      - "Yanis Guenane (@Spredzy)"
+    short_description: Vultr inventory source
+    description:
+        - Get inventory hosts from Vultr public cloud.
+        - Uses ~/.vultr.ini, ./vultr.ini or VULTR_API_CONFIG path to config file.
+    options:
+        plugin:
+            description: token that ensures this is a source file for the 'vultr' plugin.
+            required: True
+            choices: ['vultr']
+        api_account:
+            description: Specify the account to be used.
+'''
+
+EXAMPLES = r'''
+# vultr_inventory.yml file in YAML format
+# Example command line: ansible-inventory --list -i vultr_inventory.yml
+
+plugin: vultr
+'''
+
+import json
+import os
+
+from ansible.errors import AnsibleError
+from ansible.plugins.inventory import BaseInventoryPlugin
+from ansible.module_utils.six.moves import configparser
+from ansible.module_utils.urls import open_url
+from ansible.module_utils._text import to_native
+
+
+def _retrieve_api_key(api_account):
+    if os.environ.get('VULTR_API_KEY'):
+        return os.environ.get('VULTR_API_KEY')
+    else:
+        paths = []
+        if os.environ.get('VULTR_API_CONFIG'):
+            paths += (os.path.expanduser(os.environ.get('VULTR_API_CONFIG'),))
+        paths += (
+            os.path.join(os.path.expanduser('~'), '.vultr.ini'),
+            os.path.join(os.getcwd(), 'vultr.ini'),
+        )
+
+        conf = configparser.ConfigParser()
+        conf.read(paths)
+
+        if not conf._sections.get(api_account):
+            return None
+
+        return dict(conf.items(api_account)).get('key')
+
+
+def _retrieve_servers(api_key):
+    api_url = 'https://api.vultr.com/v1/server/list'
+
+    try:
+        response = open_url(
```

Comment:
Should not `'User-Agent': "Ansible Vultr"` be reused here ?

Reply:
Done.